### PR TITLE
A few final edits to the keymap and readme.

### PR DESCRIPTION
**Pull request by buzzlighter1**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xd75/keymaps/buzzlighter1/readme.md [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
+ * | _______| P1     | P2     | P3     | -      | P/     | XXXXXXX| OS     |  PLAY  | PREV   | NXT    | BL_OFF | BL_ON  | RGB VD | RGB VI |
  * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
- * | CPSLCK | P0     | P.     | _______|                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
+ * | _______| P0     | P.     | PENT   |                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
  * '--------------------------------------------------------------------------------------------------------------------------------------'
- */
 
 
-## LED control
+### LEDs ID
```
Comment (zvecr) [inline keyboards/xd75/keymaps/buzzlighter1/readme.md:37]: This info isnt really part of `## FUNCTION`
```suggestion
## LEDs ID
```

File: keyboards/xd75/keymaps/buzzlighter1/readme.md [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
-Caps Lock and Win keys sent to the Function layer.
-TT(1) - Function layer tap-toggle at CapsLock position provides fast F-keys access in a heat of a battle. As well as easy access to a Numpad.
+Standard QWERTY made for gamers with a Russian alternative input.
+The keys for extra letters in RU alphabet are where they are expected.
+Full 2x4 nav cluster. 
+Volume/mute control are on base layer, Win key is on function layer.
+FN_CAPS provides fast F-keys access while gaming. Lefthanded Numpad operation.
+NKRO is working (forced).
 
-/* QWERTY
```
Comment (zvecr) [inline keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: Rendering is kinda bad here still, you might want to revert and wrap with triple backticks,

    ```c
    /* QWERTY
     * .--------------------------------------------------------------------------------------------------------------------------------------.
     * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
    ...
     */
    ```

will produce

```c
/* QWERTY
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
...
 */
```

File: keyboards/xd75/keymaps/buzzlighter1/readme.md (hunk 1 above)
Comment (buzzlighter1) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md]: # XD75 with 7U spacebar for EN-RU gamers

Standard QWERTY made for gamers with a Russian alternative input.
The keys for extra letters in RU alphabet are where they are expected.
Full 2x4 nav cluster. 
Volume/mute control are on base layer, Win key is on function layer.
FN_CAPS provides fast F-keys access while gaming. Lefthanded Numpad operation.
NKRO is working (forced).

```c
## QWERTY
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
 * | TAB    | Q      | W      | E      | R      | T      | Y      | U      | I      | O      | P      | [      | ]      | \      | Mute   |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
 * | FN(CPS)| A      | S      | D      | F      | G      | H      | J      | K      | L      | ;      | '      | Vol-   | Vol+   | ENTER  |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
 * | LSHIFT | Z      | X      | C      | V      | B      | N      | M      | ,      | .      | /      | Home   | PGUP   | UP     | PGDWN  |
 * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
 * | LCTRL  | Del    | ENTER  | LALT   |                             SPACE                            | End    | LEFT   | DOWN   | RIGHT  |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
```

```c
## FUNCTION
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | XXXXXXX| F1     | F2     | F3     | F4     | F5     | F6     | F7     | F8     | F9     | F10    | F11    | F12    | _______| _______|
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P7     | P8     | P9     | +      | P*     | XXXXXXX| PR SCR | SCR LK | PAUSE  | XXXXXXX| BL_DEC | BL_INC | RGB HD | RGB HI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P4     | P5     | P6     | NumLk  | CALC   | XXXXXXX| POWER  | XXXXXXX| RESET  | XXXXXXX| BL_BRTG| BL_STEP| RGB SD | RGB SI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P1     | P2     | P3     | -      | P/     | XXXXXXX| OS     |  PLAY  | PREV   | NXT    | BL_OFF | BL_ON  | RGB VD | RGB VI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P0     | P.     | PENT   |                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
```

## LEDs ID

Top left LED - Function layer active.
Mid left LED - CapsLock active.

File: keyboards/xd75/keymaps/buzzlighter1/readme.md (hunk 2 above)
Comment (zvecr) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: Still outstanding.

Comment (buzzlighter1) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: # XD75 with 7U spacebar for EN-RU gamers
Standard QWERTY made for gamers with a Russian alternative input.
The keys for extra letters in RU alphabet are where they are expected.
Full 2x4 nav cluster. 
Volume/mute control are on base layer, Win key is on function layer.
FN_CAPS provides fast F-keys access while gaming. Lefthanded Numpad operation.
NKRO is working (forced).
## Layout
### Qwerty layout
```
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
 * | TAB    | Q      | W      | E      | R      | T      | Y      | U      | I      | O      | P      | [      | ]      | \      | Mute   |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
 * | FN(CPS)| A      | S      | D      | F      | G      | H      | J      | K      | L      | ;      | '      | Vol-   | Vol+   | ENTER  |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
 * | LSHIFT | Z      | X      | C      | V      | B      | N      | M      | ,      | .      | /      | Home   | PGUP   | UP     | PGDWN  |
 * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
 * | LCTRL  | Del    | ENTER  | LALT   |                             SPACE                            | End    | LEFT   | DOWN   | RIGHT  |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
```
### Function (FN)
```
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | XXXXXXX| F1     | F2     | F3     | F4     | F5     | F6     | F7     | F8     | F9     | F10    | F11    | F12    | _______| _______|
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P7     | P8     | P9     | +      | P*     | XXXXXXX| PR SCR | SCR LK | PAUSE  | XXXXXXX| BL_DEC | BL_INC | RGB HD | RGB HI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P4     | P5     | P6     | NumLk  | CALC   | XXXXXXX| POWER  | XXXXXXX| RESET  | XXXXXXX| BL_BRTG| BL_STEP| RGB SD | RGB SI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P1     | P2     | P3     | -      | P/     | XXXXXXX| OS     |  PLAY  | PREV   | NXT    | BL_OFF | BL_ON  | RGB VD | RGB VI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P0     | P.     | PENT   |                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
```
## LEDs ID
Top left LED - Function layer active.
Mid left LED - CapsLock active.

Comment (zvecr) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: nope, see <https://github.com/qmk/qmk_firmware/blob/db4b11f866477791c15de75a72d6444bdb0e6449/keyboards/xd75/keymaps/buzzlighter1/readme.md>

Comment (buzzlighter1) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: Do you have an example with proper formatting?

Comment (zvecr) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: <https://github.com/zvecr/qmk_firmware/blob/1bc30a161fd41b96052588e702f88fe1b134cf44/keyboards/xd75/keymaps/buzzlighter1/readme.md>

Comment (buzzlighter1) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: # XD75 with 7U spacebar for EN-RU gamers
 * Standard QWERTY made for gamers with a Russian alternative input.
 * The keys for extra letters in RU alphabet are where they are expected.
 * Full 2x4 nav cluster.
 * Volume/mute control are on base layer, Win key is on function layer.
 * FN_CAPS provides fast F-keys access while gaming. Lefthanded Numpad operation.
 * NKRO is on (forced).

## Layout
```c
/* QWERTY
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
 * | TAB    | Q      | W      | E      | R      | T      | Y      | U      | I      | O      | P      | [      | ]      | \      | Mute   |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
 * | FN(CPS)| A      | S      | D      | F      | G      | H      | J      | K      | L      | ;      | '      | Vol-   | Vol+   | ENTER  |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
 * | LSHIFT | Z      | X      | C      | V      | B      | N      | M      | ,      | .      | /      | Home   | PGUP   | UP     | PGDWN  |
 * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
 * | LCTRL  | Del    | ENTER  | LALT   |                             SPACE                            | End    | LEFT   | DOWN   | RIGHT  |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
 */
```
```c
/* FUNCTION
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | XXXXXXX| F1     | F2     | F3     | F4     | F5     | F6     | F7     | F8     | F9     | F10    | F11    | F12    | _______| _______|
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P7     | P8     | P9     | +      | P*     | XXXXXXX| PR SCR | SCR LK | PAUSE  | XXXXXXX| BL_DEC | BL_INC | RGB HD | RGB HI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P4     | P5     | P6     | NumLk  | CALC   | XXXXXXX| POWER  | XXXXXXX| RESET  | XXXXXXX| BL_BRTG| BL_STEP| RGB SD | RGB SI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P1     | P2     | P3     | -      | P/     | XXXXXXX| OS     |  PLAY  | PREV   | NXT    | BL_OFF | BL_ON  | RGB VD | RGB VI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P0     | P.     | PENT   |                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
 */
```
## LEDs
Top left LED - Function layer active.
Mid left LED - CapsLock active.

## Compile

go to qmk top directory.
```
$ cd qmk_firmware
```

build
```
$ make xd75:buzzlighter1
```

Comment (zvecr) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: Yeah thats what it looks like fixed with my proposed changes.

Your current instance does not render correctly on github within firefox or chrome, windows and linux, can we just make the changes and move on please.

Comment (buzzlighter1) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:9]: I'm sorry, is the latest version OK or not? What should I do now?
Previously I was just accepting edits. Now I can only see and option to comment (that is why I have sent the file as a comment).

File: keyboards/xd75/keymaps/buzzlighter1/readme.md [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
+## QWERTY
```
Comment (zvecr) [inline keyboards/xd75/keymaps/buzzlighter1/readme.md:10]:
````suggestion
## QWERTY

```c
````

File: keyboards/xd75/keymaps/buzzlighter1/readme.md [hunk 4]
```diff
@@ -1,41 +1,40 @@
...
  * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
  * | LSHIFT | Z      | X      | C      | V      | B      | N      | M      | ,      | .      | /      | Home   | PGUP   | UP     | PGDWN  |
  * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
- * | LCTRL  | Prev   | Next   | LALT   |                                SPACE                         | End    | LEFT   | DOWN   | RIGHT  |
+ * | LCTRL  | Del    | ENTER  | LALT   |                             SPACE                            | End    | LEFT   | DOWN   | RIGHT  |
  * '--------------------------------------------------------------------------------------------------------------------------------------'
- */
 
-/* FUNCTION
+## FUNCTION
```
Comment (zvecr) [inline keyboards/xd75/keymaps/buzzlighter1/readme.md:23]:
````suggestion
```

## FUNCTION

```c
````

File: keyboards/xd75/keymaps/buzzlighter1/readme.md [hunk 5]
```diff
@@ -1,41 +1,40 @@
...
+ * | _______| P4     | P5     | P6     | NumLk  | CALC   | XXXXXXX| POWER  | XXXXXXX| RESET  | XXXXXXX| BL_BRTG| BL_STEP| RGB SD | RGB SI |
  * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
- * | P/     | P1     | P2     | P3     | -      | OS     | XXXXXXX| XXXXXXX| XXXXXXX| XXXXXXX| XXXXXXX| BL_OFF | BL_ON  | RGB VD | RGB VI |
+ * | _______| P1     | P2     | P3     | -      | P/     | XXXXXXX| OS     |  PLAY  | PREV   | NXT    | BL_OFF | BL_ON  | RGB VD | RGB VI |
  * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
- * | CPSLCK | P0     | P.     | _______|                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
+ * | _______| P0     | P.     | PENT   |                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
  * '--------------------------------------------------------------------------------------------------------------------------------------'
- */
```
Comment (zvecr) [inline keyboards/xd75/keymaps/buzzlighter1/readme.md:35]:
````suggestion
```

````

Comment (buzzlighter1) [reply keyboards/xd75/keymaps/buzzlighter1/readme.md:35]: As I understand it does not let you copypaste over the entire file. I will close this PR and open a new one then if you don't mind.

**Comment by drashna** [comment]
Thanks. 

For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

**You do not need to close this pull request at this time.** Until your pull request is merged, please continue to commit relevant changes to your `master` branch.

**Pull request merged**